component/client/core/ClientCore.tsx

```diff
@@ -101,8 +101,8 @@ class ClientCore extends React.Component<any, ClientCoreState> {
 
   private handleSave(): void {
     storage.set(STORAGE_KEY_USERNAME, this.state.username)
-    .then(storage.set(STORAGE_KEY_INITIALIZED, true)
-      .then(this.setState({initialized: true, username: ""})));
+      .then(() => storage.set(STORAGE_KEY_INITIALIZED, "true")
```

Comment:
I don't think `value` should be a string because according to the docs (https://developer.chrome.com/extensions/storage), the `value` just needs to be an object. Unless we are going to limit ourselves to only storing string values, I don't think that typing should be restricted.